Sources/SwifterSwift/UIKit/UILabelExtensions.swift

```diff
@@ -9,7 +9,56 @@
 #if canImport(UIKit) && !os(watchOS)
 import UIKit
 
+// MARK: - Properties
+public extension UILabel {
+
+    /// SwifterSwift: Required height for a label
+    var requiredHeight: CGFloat {
+        let label = UILabel(frame: CGRect(x: 0, y: 0, width: frame.width, height: CGFloat.greatestFiniteMagnitude))
```

Comment:
I'm not sure about this... I think creating another label object is not the best way to move this forward.
Is there any way to calculate this only based on text and font size (maybe some other system information e.g. dynamic size)?

Reply:
Oh I just noted this was already here, sorry 😄

Reply:
We probably need to revisit this extension ... cc @omaralbeik

Reply:
> We probably need to revisit this extension ... cc @omaralbeik

Definitely, there are lots of UIKit extensions that can benefit from a revisit 💯